### ISSUE TITLE
[Accessibility]: Multiple toolbar components do not have unique labels

### ISSUE THREAD

**Issue by Akshat55** [carbon-design-system/carbon-charts#1390]
### Environment

Browser, Assistive technology used to verify

### What happened? What did you expect to happen instead?

When there are two or more charts with toolbar's enabled, the toolbar role needs to have an aria-label or aria-labelledby attribute (in this case, since there is no text - it should be aria-label)

![image](https://user-images.githubusercontent.com/38994122/173471220-cd3eb5b1-6e54-4e36-927c-955ab7954a66.png)

### What WCAG 2.1 checkpoint does the issue violate?

4.1.2 Name, Role, Value

### Version

@carbon/charts@1.2.0

### Data & options used

```js
When there are two charts on the page, used IBM Accessibility Assessment
```

### Codesandbox example

https://codesandbox.io/s/yu4jyv
